app/assets/stylesheets/_globals.sass

```diff
@@ -19,69 +19,112 @@ header
       margin: 15px auto 0
       display: block
 
+.active-item
+  border: 3px solid $feed-item-border-color
+  color: $application-text-color
+  opacity: 1
+  .options
+    .option
+      label
+        color: $option-color
+        &:hover
+          background-color: $option-background-color-hover
+          color: $option-color-hover
+          cursor: pointer
+
+      input[type="radio"]:checked
+        & + label
+          background-color: $option-background-color-checked
+          box-shadow: inset 0 1px 6px $option-inset-shadow-color-checked
+          color: $option-color-checked
+
+.disabled-item
```

Comment:
Ik zie nergens .disabled-item in de view voorkomen. Bedoelde je `.item.disabled`?